app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAboutRobot.kt

```diff
@@ -96,106 +112,102 @@ private fun assertCurrentTimestamp() {
-
-    onView(withId(R.id.mozac_browser_toolbar_title_view)).check(
-        matches(
-            withText(
-                containsString("What's new in Firefox Preview")
-            )
-        )
+    TestHelper.verifyUrl(
+        SupportUtils.SumoTopic.WHATS_NEW.topicStr,
+        "org.mozilla.fenix.debug:id/mozac_browser_toolbar_url_view",
+        R.id.mozac_browser_toolbar_url_view
     )
+
     Espresso.pressBack()
 }
 
 private fun assertSupport() {
-    TestHelper.scrollToElementByText("Support")
+    if (!onView(withText("Support")).isVisibleForUser()) {
+        onView(withId(R.id.about_layout)).perform(ViewActions.swipeUp())
+    }
+
     onView(withText("Support"))
-        .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+        .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
         .perform(click())
 
-    // This is used to wait for the webpage to fully load
-    TestHelper.waitUntilObjectIsFound("org.mozilla.fenix.debug:id/mozac_browser_toolbar_title_view")
-
-    onView(withId(R.id.mozac_browser_toolbar_title_view)).check(
-        matches(
-            withText(
-                containsString("Firefox Preview Help")
-            )
-
-        )
+    TestHelper.verifyUrl(
+        "support.mozilla.org",
+        "org.mozilla.fenix.debug:id/mozac_browser_toolbar_url_view",
+        R.id.mozac_browser_toolbar_url_view
     )
+
     Espresso.pressBack()
 }
 
 private fun assertPrivacyNotice() {
-    TestHelper.scrollToElementByText("Privacy Notice")
+    if (!onView(withText("Privacy notice")).isVisibleForUser()) {
+        onView(withId(R.id.about_layout)).perform(ViewActions.swipeUp())
+    }
+
     onView(withText("Privacy notice"))
-        .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+        .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
         .perform(click())
 
-    // This is used to wait for the webpage to fully load
-    TestHelper.waitUntilObjectIsFound("org.mozilla.fenix.debug:id/mozac_browser_toolbar_title_view")
-
-    onView(withId(R.id.mozac_browser_toolbar_title_view)).check(
-        matches(
-            withText(
-                containsString("Firefox Privacy Notice")
-            )
-
-        )
+    TestHelper.verifyUrl(
+        "/privacy/firefox",
+        "org.mozilla.fenix.debug:id/mozac_browser_toolbar_url_view",
+        R.id.mozac_browser_toolbar_url_view
     )
+
     Espresso.pressBack()
 }
 
 private fun assertKnowYourRights() {
-    TestHelper.scrollToElementByText("Know your rights")
+    if (!onView(withText("Know your rights")).isVisibleForUser()) {
+        onView(withId(R.id.about_layout)).perform(ViewActions.swipeUp())
+    }
+
     onView(withText("Know your rights"))
-        .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+        .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
```

Comment:
Is there a reason we're using `isVisibleForUser` in some cases an not others like this?

Reply:
It is not necessarily to use it here because it is clearly that the view is 100% visible since we already checked above that it is completely visible (check the comment below).